### PR TITLE
Refactor analyzer log to show the actually used queries

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/piped/executor/analysis/metrics_analyzer.go [hunk 1]
```diff
@@ -132,6 +132,8 @@ func (a *metricsAnalyzer) analyzeWithThreshold(ctx context.Context) (bool, error
 		From: now.Add(-a.cfg.Interval.Duration()),
 		To:   now,
 	}
+
+	a.logPersister.Infof("[%s] Run query: %q, in range: %s", a.cfg.Query, queryRange)
```
Comment (knanao) [inline pkg/app/piped/executor/analysis/metrics_analyzer.go:136]: Should we add the analyzer id into params?

Comment (khanhtc1202) [inline pkg/app/piped/executor/analysis/metrics_analyzer.go:136]: looks like the params are mismatched?

Comment (nghialv) [reply pkg/app/piped/executor/analysis/metrics_analyzer.go:136]: Ah right. 🤣

Comment (nghialv) [reply pkg/app/piped/executor/analysis/metrics_analyzer.go]: Fixed it!

Comment (nghialv) [reply pkg/app/piped/executor/analysis/metrics_analyzer.go]: Fixed it!

**Review by knanao — APPROVED** [review]
Nice catch!

**Review by khanhtc1202 — APPROVED** [review]
🚀

**Pull request merged**